### ISSUE TITLE
Created WorkItem don't' show up in the kanban board (VSTS)

### ISSUE THREAD

**Issue by marcoCasamento** [Microsoft/mail2bug#61]
As per the title it seems that the Workitem created by mail2bug don't show up in the kanban board of VSTS. any idea ?

**Comment by hershi** [comment]
I've never used VSTS Kanban board, so can't say for sure but:
1. Mail2Bug created items have nothing special to them - mail2bug is written as an external tool using public APIs to query, create, and update work items, so they should behave like any other work item
2. My best guess would be that the Kanban board is using some type of query to determine which items to show, and that the default values you configured for items do not match that query, resulting in them not being included. That being said, if anyone with better knowledge/understanding of the VSTS Kanban board can corroborate this or offer more/other insights, that would be better

To summarize, though, I doubt that this has anything to do with Mail2Bug code, and probably more related to the specific configuration settings

**Comment by marcoCasamento** [comment]
Ok, thanks, I've sent an email to vsointegration@microsoft.com I'll update/make a PR if ever found a solution

**Comment by marcoCasamento** [comment]
Ok I've got an helpful answer from them and it has nothing to do with mail2bug, I'm reporting it here for the sake of completeness:

> Backlogs and boards are filtered view of work items.
> The filter is based on Area path, Work Item Type, State and Iteration path.
> If you go the corresponding backlog of the board and select “Create Query” you will see the filters we use.
> Setting the field to right value will make the work item available in the board.
> 

Simply I put the wrong value for the Area Path field.
